dotnet/test/common/DevTools/DevToolsProfilerTest.cs

```diff
@@ -131,12 +131,12 @@ private void ValidateProfile(CurrentCdpVersion.Profiler.Profile profiler)
         {
             Assert.That(profiler, Is.Not.Null);
             Assert.That(profiler.Nodes, Is.Not.Null);
-            Assert.That(profiler.StartTime, Is.Not.Null);
-            Assert.That(profiler.EndTime, Is.Not.Null);
+            Assert.That(profiler.StartTime, Is.Not.Zero);
```

Comment:
Modern NUnit comes with some analyzers which explained that `profiler.StartTime`, `profiler.EndTime`, and `delta` are value types (all doubles) so can never be null. These tests pass if we assert instead that they are `!= 0`

Reply:
Not clear what zero means. If we are context of datetimes, let's try too be specific.

Reply:
So I took a deeper look at this, and although it's storing datetime values, it does so as unix timestamps (and therefore, numbers).

![image](https://github.com/user-attachments/assets/e5838dfa-2831-4950-afe9-6f2b283932f5)
![image](https://github.com/user-attachments/assets/b7c9ed17-ac6a-4685-aa10-f0b383528f6d)

NUnit 4 introduces an `Is.(Not.)Default` assertion, which is all we're really testing here (making sure the field is populated), but we don't have access to that yet. Either way, I hope with this clarification we can accept `Is.Not.Zero` as a meaningful assertion. Let me know if there's something else we should do (maybe parse to DateTime from unit timestamp?)

Reply:
Thanks.